app/views/settings/index.html.erb

```diff
@@ -17,7 +17,7 @@
 
     <%= f.field(:incoming_sms_numbers_str, type: :textarea) %>
 
-    <%= f.regenerable_field(:incoming_sms_token,
+    <%= f.regenerable_field(:incoming_sms_token, confirm: t('setting.sms_token_warning'),
```

Comment:
Can you plz refactor all in this file to double quotes? And do this as a rule when touching old files?

Reply:
Can you also please refactor all `using-` as a CSS class (like below line) to not use underscores? So `using-incoming-sms-token`, `using-universal-sms-token`, etc. Trying to slowly get rid of underscores in CSS classes.